### PR TITLE
Flatten video props

### PR DESCRIPTION
**Pull request by jtart**
_Removes `model` prop and just pass `blocks`. Also updates the fixture data and props that is used._

* [x] Fixes https://github.com/bbc/simorgh/issues/363
* [x] Tests added for new features

* [ ] ~~Test engineer approval~~

### CODE REVIEW

**Review by radiocontrolled — APPROVED** [review]

File: src/app/models/propTypes/index.js [hunk 1]
```diff
@@ -41,27 +41,25 @@ export const imagePropTypes = {
...
-              locator: string.isRequired,
-            }),
+  blocks: arrayOf(
+    shape({
+      locator: string,
+      blocks: arrayOf(
+        // raw video
+        shape({
+          model: shape({
+            isLive: bool,
```
Comment (DemonusPC) [inline src/app/models/propTypes/index.js:51]: Shouldn't these all be required for the BBC player to work properly?

File: src/app/models/propTypes/index.js [hunk 2]
```diff
@@ -41,27 +41,25 @@ export const imagePropTypes = {
...
-          // raw video
-          shape({
-            model: shape({
-              isLive: bool,
-              duration: string,
-              locator: string.isRequired,
-            }),
+  blocks: arrayOf(
+    shape({
+      locator: string,
```
Comment (DemonusPC) [inline src/app/models/propTypes/index.js:46]: I think the locator is the same as the locator in the raw video block so technically should be required as well

**Review by DemonusPC — CHANGES_REQUESTED** [review]

File: src/app/models/propTypes/index.js (hunk 1 above)
Comment (jtart) [reply src/app/models/propTypes/index.js:51]: Possibly, but that is outside the remit of this PR really. Intend to open more issues to review props of Image and Video, and move them to the new format.

File: src/app/models/propTypes/index.js (hunk 2 above)
Comment (jtart) [reply src/app/models/propTypes/index.js:46]: Possibly, but that is outside the remit of this PR really. Intend to open more issues to review props of Image and Video, and move them to the new format.

**Review by DemonusPC — APPROVED** [review]
Alright then. Looks good.

**Pull request merged**